volumes/ecocounter/data_checks/select-ongoing_outages.sql

```diff
@@ -1,54 +1,45 @@
-WITH out_of_order AS (
+WITH ooo_sites AS (
+    SELECT
+        site_id,
+        site_description,
+        last_active
+    FROM ecocounter.sites
+    WHERE
+        last_active < CURRENT_DATE - 1
```

Comment:
I changed to `CURRENT_DATE - 1` (recommended by postgres docs 👀) instead of basing it off of `ds`, because this way it will be more accurate if task is rerun on a different date.

Reply:
makes sense!